samples/configureRetries.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {
+  IdempotencyStrategy,
+} = require('@google-cloud/storage/build/src/storage');
+
+/**
+ * This application demonstrates how to perform basic operations on buckets with
+ * the Google Cloud Storage API.
+ *
+ * For more information, see the README.md under /storage and the documentation
+ * at https://cloud.google.com/storage/docs.
+ */
+
+function main() {
+  // [START storage_configure_retries]
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage({
+    retryOptions: {
+      autoRetry: true, //If this is false, requests will not retry and the setting of the below parameters will not have any effect.
```

Comment:
"and the parameters below will not affect the retry behavior"
Can also use "settings" instead of "behavior" if it makes more sense, but the doc currently use "behavior"

Reply:
Just a nit - can we change the formatting of the comments so it doesn't extend too far in the code block and become difficult to scroll? 

For example:
```
// If this is false, requests will not retry and the
// parameters below will not affect the retry behavior.
autoRetry: true,
// The multiplier by which...
// ...
retryDelayMultiplier: 3
```
I'm happy to make these changes myself in a follow up PR so they aren't a blocker.